src/plugins/intel_cpu/tests/functional/shared_tests_instances/behavior/compiled_model/import_export.cpp

```diff
@@ -23,6 +26,57 @@ const std::vector<ov::element::Type_t> netPrecisions = {
 };
 const ov::AnyMap empty_property = {};
 
+using ov::op::v0::Parameter, ov::op::v0::Result;
+
+TEST_P(OVCompiledGraphImportExportTest, importExportModelWithTypeRelaxedExtension) {
+    // Create model with interpolate which v0 and v4 are supported by TypeRelaxedExtension
```

Comment:
As I understand, the original issue was about Interpolate 11
can we cover this case also?
<img width="156" alt="{4DE78D10-A115-4644-A142-437C229A5C6F}" src="https://github.com/user-attachments/assets/21a63adb-edd0-4001-9cab-2b9c393918fc" />

Reply:
Original issue was for Interpolate v4 which on import was constructed as v0, incompatible input.
v11 is not supported in TypeRelaxedExtension.

Reply:
ok, thanks